services/src/main/resources/locale/addon/Gamification_en.properties

```diff
@@ -23,6 +23,16 @@ exoplatform.gamification.leaderboard.rank=Your current rank
 exoplatform.gamification.leaderboard.Pie=Pie chart loading
 exoplatform.gamification.leaderboard.Pieopen=Open Pie chart
 #Gamification Informations Portlet#
+exoplatform.gamification.gamificationinformation.rule.title.CreateEvent=Agenda : Create an event
```

Comment:
no agenda in Meeds-io, so you may need to add those labels in agenda instead of here (with the same path to extend its content)